src/layer/Tooltip.js

```diff
@@ -413,9 +413,18 @@ Layer.include({
 
 
 	_openTooltip(e) {
-		if (!this._tooltip || !this._map || (this._map.dragging && this._map.dragging.moving())) {
+		if (!this._tooltip || !this._map) {
 			return;
 		}
+
+		// If the map is moving, we will show the tooltip after it's done.
+		if (this._map.dragging && this._map.dragging.moving()) {
+			this._map.once('moveend zoomend', () => {
+				this._openTooltip(e);
```

Comment:
Calling `_openTooltip` here is to early. the `dragging.moving()` function will be still set to `true`. This needs a change in `Draggable.js`, the flag needs to be set to `false` before fireing the event.

```

		if (this._moved && this._moving) {

			// @event dragend: DragEndEvent
			// Fired when the drag ends.
			this.fire('dragend', {
				noInertia,
				distance: this._newPos.distanceTo(this._startPos)
			});
		}

		this._moving = false;
		Draggable._dragging = false;
```

to:
```
		const fireDragend = this._moved && this._moving;

		this._moving = false;
		Draggable._dragging = false;

		if (fireDragend) {
			// @event dragend: DragEndEvent
			// Fired when the drag ends.
			this.fire('dragend', {
				noInertia,
				distance: this._newPos.distanceTo(this._startPos)
			});
		}
```